src/de/2024-04/12/teacher-questions.md

```diff
@@ -11,71 +11,72 @@ date:   20/12/2024
 
 ## STARTFRAGEN
 
--   ...
+-   -   …
 
-## SONNTAG -- Was ist Wahrheit?
+## SONNTAG – Was ist Wahrheit?
 
-**Joh 18,33-38**
+'''Joh 18,33-38““
 
 -   Fragen zum Text:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 -   Fragen zur Botschaft:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 
-## MONTAG -- Siehe, der Mensch!
+## MONTAG – Siehe, der Mensch!
 
-\'\'\'Joh 18,38-19,5""
+'''Joh 18,38-19,5““
 
 -   Fragen zum Text:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 -   Fragen zur Botschaft:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 
-## DIENSTAG -- Es ist vollbracht
+## DIENSTAG – Es ist vollbracht
 
 **Joh 19,25-30**
 
 -   Fragen zum Text:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 -   Fragen zur Botschaft:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 
-## MITTWOCH -- Das leere Grab
+## MITTWOCH – Das leere Grab
 
-\'\'\'Joh 20,1-10'''
+'''Joh 20,1-10‘‘‘
```

Comment:
@toak This looks to me like a misformatted markdown. 
1. What is that supposed to be rendered at?
2. The closing three symbols are different that opening

Reply:
that was an issue in the data source, it's fixed now